rosalind_ini3.py

```diff
@@ -0,0 +1,7 @@
+def main(input):
+    (corpus, params) = input.strip().split('\n')
+    (word_one_start, word_one_end,
+     word_two_start, word_two_end) = map(int, params.split(' '))
```

Comment:
Okay these variable assignments make sense to me after looking up map().

Also, just testing what the inline comments are like.